### PR TITLE
add bookmark page & button that toggles isBookmarked

### PR DESCRIPTION
**Pull request by TariqAlQadi**
created bookmark page, button that toggles color and isBookmarked state

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -29,7 +31,9 @@ export default function App({ Component, pageProps }) {
       <Head>
         <title>Capstone Project</title>
       </Head>
+      <Header />
       <Component {...pageProps} />
+      <Navigation />
     </SWRConfig>
```
Comment (SaschaRadojkovic) [inline pages/_app.js:37]: how far i see there is only a closing </SWRConfig> and it is not warpping the component tag

File: pages/tutorials/index.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import CardList from "@/components/CardList";
-import Header from "@/components/Header";
 import useSWR from "swr";
```
Comment (SaschaRadojkovic) [inline pages/tutorials/index.js:2]: useSWR is imported but not used mabe you should remove it

File: pages/_app.js (hunk 1 above)
Comment (TariqAlQadi) [reply pages/_app.js:37]: Hi Sascha 
have a closer look .. there's an opening tag aswell .. otherwise the application would not work 😇

Comment (TariqAlQadi) [reply pages/_app.js:37]: I just added the Navigation and the SWRConfig opening tag is not in view, because its above the Head component

Comment (SaschaRadojkovic) [reply pages/_app.js:37]: ah sorry i did not see that it was not visible ;)

Comment (TariqAlQadi) [reply pages/_app.js:37]: no problemo!

File: components/CardDetails/index.js [hunk 3]
```diff
@@ -12,7 +24,18 @@ export default function CardDetails({ content }) {
...
+      <button type="button" onClick={handleToggleBookmark}>
+        {isBookmarked ? (
+          <svg viewBox="0 0 384 512" width="20px" height="20px">
+            <path d="M0 48V487.7C0 501.1 10.9 512 24.3 512c5 0 9.9-1.5 14-4.4L192 400 345.7 507.6c4.1 2.9 9 4.4 14 4.4c13.4 0 24.3-10.9 24.3-24.3V48c0-26.5-21.5-48-48-48H48C21.5 0 0 21.5 0 48z" />
+          </svg>
+        ) : (
+          <svg viewBox="0 0 384 512" width="20px" height="20px">
+            <path d="M336 0h-288C21.49 0 0 21.49 0 48v431.9c0 24.7 26.79 40.08 48.12 27.64L192 423.6l143.9 83.93C357.2 519.1 384 504.6 384 479.9V48C384 21.49 362.5 0 336 0zM336 452L192 368l-144 84V54C48 50.63 50.63 48 53.1 48h276C333.4 48 336 50.63 336 54V452z" />
+          </svg>
+        )}
```
Comment (doemser) [inline components/CardDetails/index.js:37]: Try building a cutsom SVG component like this:

https://codesandbox.io/s/dreamy-joana-13ij6o?file=/src/App.js

**Review by doemser — CHANGES_REQUESTED** [review]

File: pages/details/[id].js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  const [list, setList] = useAtom(beginnerList);
+
+  const router = useRouter();
+  const { id } = router.query;
+
+  function handleToggleLike(id) {
+    setList(
+      list.map((tutorial) => {
+        if (tutorial.id === id) {
+          console.log(tutorial);
```
Comment (SaschaRadojkovic) [inline pages/details/[id].js:16]: here is a console.log yet

**Pull request merged**